docker-compose.yml

```diff
@@ -33,3 +35,23 @@ services:
     image: "redis"
     ports:
       - "6379:6379"
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.2
+    container_name: elasticsearch
+    environment:
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "discovery.type=single-node"
```

Comment:
May want to change this in the future to allow for multiple nodes but this is good for now.

Reply:
Yes, I thought about that, but figured let's keep it simple for now, we shouldn't need bigger clusters for the dev workflow.